apis/submariner/v1alpha1/submariner_types.go

```diff
@@ -49,10 +49,16 @@ type SubmarinerSpec struct {
 	Version                  string `json:"version,omitempty"`
 	CeIPSecIKEPort           int    `json:"ceIPSecIKEPort,omitempty"`
 	CeIPSecNATTPort          int    `json:"ceIPSecNATTPort,omitempty"`
-	CeIPSecDebug             bool   `json:"ceIPSecDebug"`
-	Debug                    bool   `json:"debug"`
-	NatEnabled               bool   `json:"natEnabled"`
-	ServiceDiscoveryEnabled  bool   `json:"serviceDiscoveryEnabled,omitempty"`
+	// The interval in seconds in which the HealthChecker ping will be be send
+	HealthCheckInterval uint64 `json:"healthCheckInterval,omitempty"`
+	// The interval for which the HealthChecker will wait before marking a
+	// connection as down
```

Comment:
Maybe change to something like "Defines how long we should wait for HealthChecker ping packets before we declare the connection as down?" Also, is this in seconds?

Reply:
Done.